ansible/playbooks/site.yml

```diff
@@ -1,5 +1,8 @@
 ---
-- ansible.builtin.import_playbook: db.yml
-- ansible.builtin.import_playbook: app.yml
-- ansible.builtin.import_playbook: deploy.yml
-- ansible.builtin.import_playbook: users.yml
+- name: Install MongoDB VM
+  ansible.builtin.import_playbook: db.yml
+- name: Install Reddit monolith VM
+  ansible.builtin.import_playbook: app.yml
+- name: Install Reddit application
+  ansible.builtin.import_playbook: deploy.yml
+# - ansible.builtin.import_playbook: users.yml
```

Comment:
If you want to remove your code it is better to remove it rather than comment because you can accidentally uncomment this and get an error